src/client/tile.cpp

```diff
@@ -2266,7 +2265,7 @@ video::ITexture* TextureSource::getNormalTexture(const std::string &name)
 	if (isKnownSourceImage("override_normal.png"))
 		return getTexture("override_normal.png");
 	std::string fname_base = name;
-	std::string normal_ext = "_normal.png";
+	static const std::string normal_ext = "_normal.png";
```

Comment:
in fact the interesting keyword is static to prevent reallocating this variable each time we do a getNormalTexture, just grab the global ref, but i can remove this optimisation if you doesn't want